src/modules/powerrename/ui/PowerRenameUI.cpp

```diff
@@ -927,6 +941,50 @@ void CPowerRenameUI::_UpdateCounts()
     }
 }
 
+void CPowerRenameUI::_CollectItemPosition(_In_ DWORD id)
```

Comment:
Just to keep code bit more readable, You can compute `rcWindow.right - rcWindow.left` and `rcWindow.bottom - rcWindow.top` on the beginning and use it later in switch-cases (`width` and `height`) in the same manner as You've did above in `_MoveControl`.

Reply:
Nice

Reply:
Done